docs/setup-cluster/deploy-cluster/slurm/slurm-known-issues.rst

```diff
@@ -567,3 +567,51 @@ allocation on each node. Your ``slots_per_trial`` configuration should then be a
    commands will print an error message.
 
 -  ``det slot list`` will not display the name of any active Determined tasks.
+
+.. _hpc_package_verification:
+
+**********************
+ Package Verification
+**********************
+
+The launcher installation package supports the verification of both RPM and DEB packages. There will
+be several configuration files that the package manager will identify as modified, and with
+RPM-based installs, some files will show user/group modifications.
+
+For an RPM-based installation, run ``sudo rpm -V hpe-hpc-launcher`` which should produce output
+similar to that shown below:
```

Comment:
"The launcher installation package can verify both RPM and DEB packages. Some configuration files will be identified as modified by the package manager. For RPM installs, certain files will also show user/group changes.

To verify an RPM installation, run ``sudo rpm -V hpe-hpc-launcher``. The output should resemble the following:"